### PR TITLE
roachtest: skip crdb/{crdb,sink}-chaos on 2.1 and 19.1

### PR DESCRIPTION
**Pull request by danhhz**
They're known to be flaky until #36852 gets backported but that won't
happen for another week.

Release note: None

### CODE REVIEW

**Comment by danhhz** [comment]
Note to self: don't merge this until after #37006 lands

**Review by tbg — APPROVED** [review]

**Comment by danhhz** [comment]
TFTR!

bors r=tbg

**Comment by danhhz** [comment]
flake looks like #35044

bors r=tbg

**Pull request merged**
